src/odemis/acq/align/transform.py

```diff
@@ -22,8 +22,9 @@
 
 from __future__ import division
 
-import numpy
```

Comment:
If you don't do any real change to this file, I'd recommend to not touch it at all.